.github/workflows/macos.yml

```diff
@@ -0,0 +1,68 @@
+name: macOS
+
+on: [push, pull_request]
+
+jobs:
+  macOS:
+    runs-on: macOS-10.15
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install system dependencies
+        run: |
+          brew install gnu-time opam gtksourceview3 adwaita-icon-theme
+          pip3 install macpack
+
+      - name: Install OCaml dependencies
+        run: |
+          export PKG_CONFIG_PATH=/usr/local/opt/libffi/lib/pkgconfig
+          opam init -a -j "$NJOBS" --compiler=ocaml-base-compiler.$COMPILER
+          opam switch set ocaml-base-compiler.$COMPILER
+          eval $(opam env)
+          opam update
+          opam install -j "$NJOBS" ocamlfind${FINDLIB_VER} ounit lablgtk3-sourceview3 zarith.1.10
+          opam list
+        env:
+          COMPILER: "4.11.1"
+          FINDLIB_VER: ".1.8.1"
+          OPAMYES: "true"
+          MACOSX_DEPLOYMENT_TARGET: "10.11"
```

Comment:
I've repeated this variable in every step where it *could* make sense, but maybe some instances could be removed.